### ISSUE TITLE
Game Tree 3-bet onwards size indication is irritating

### ISSUE THREAD

**Issue by maosatgithub** [bupticybee/TexasSolver#105]
The raise size indicated in the GUI's Game-Tree seems to be erroneous, at least I don't think it is a good way to represent it.

From the 2nd raise (3-bet) onwards, it indicates the sum of the previous raise increase + the current increase. I believe it should either indicated the total combined raise size starting with the first bet, or just the last increase.

So, if we have a bet of 5, raise to total 10, and a 3-bet to total 15, that is indicated as
BET 5, RAISE 10, RAISE 10.

I think it should be indicated either as
BET 5, RAISE 5, RAISE 5,
or
BET 5, RAISE 10, RAISE 15.

Example: (you can see at the Fold EV that the total pot size at that point was 45)
[output_parameters-raise-size-indication-bug.txt](https://github.com/bupticybee/TexasSolver/files/8793236/output_parameters-raise-size-indication-bug.txt)

![image](https://user-images.githubusercontent.com/103132228/170865901-dded0e43-d161-4740-bb83-4d0338b2b36b.png)

**Comment by SynAckFin** [comment]
The  "RAISE X" represents the amount of chips being added to the pot. So in your example the BET 5 shows that the player is adding 5 chips to the pot. The RAISE 10 means they are adding 10, and the final RAISE 10 means the the original better is adding another 10 to the pot taking their total to 15.

**Comment by bupticybee** [comment]
> The "RAISE X" represents the amount of chips being added to the pot. So in your example the BET 5 shows that the player is adding 5 chips to the pot. The RAISE 10 means they are adding 10, and the final RAISE 10 means the the original better is adding another 10 to the pot taking their total to 15.

I see, so the current implement is wrong, right?

**Comment by SynAckFin** [comment]
> 
> I see, so the current implement is wrong, right?

Not necessarily. As long as it is consistent in what it does it doesn't matter whether the "RAISE X" shows the total raise, the raise increment or the chips being added. That is just a personal preference.

As a side issue, I've been looking at the isomorphism code and in the `PCfrSolver::findGameSpecificIsomorphisms` function lines 694 -> 796 don't appear to do anything. It calculates a range_hash but never does anything with it. I would have raised a ticket but it doesn't affect any calculations.

**Comment by bupticybee** [comment]
> > I see, so the current implement is wrong, right?
> 
> Not necessarily. As long as it is consistent in what it does it doesn't matter whether the "RAISE X" shows the total raise, the raise increment or the chips being added. That is just a personal preference.
> 
> As a side issue, I've been looking at the isomorphism code and in the `PCfrSolver::findGameSpecificIsomorphisms` function lines 694 -> 796 don't appear to do anything. It calculates a range_hash but never does anything with it. I would have raised a ticket but it doesn't affect any calculations.

Oh, about that, yes, I planned to do some more isomorphism stuff there, and it didn't work out, and I guess I forget to delete all the code.
